core/tests/protractor_desktop/navigation.js

```diff
@@ -139,6 +139,15 @@ describe('Meta Tags', function() {
   });
 });
 
+describe('DEV MODE Test', function() {
+  it('should not show dev mode indication', function() {
```

Comment:
Please rephrase it to `..Dev Mode label`.

Reply:
Done. Thanks